src/components/HlsScreenSharePage.jsx

```
@@ -0,0 +1,379 @@
+        {isServerRunning && serverInfo && (
+          <>
+            <div className="code-section" style={{ marginTop: '20px' }}>
+              <p><strong>Access Code:</strong></p>
+              <div className="code-display">{serverInfo.code}</div>
+              <button 
+                onClick={() => navigator.clipboard.writeText(serverInfo.code)}
+                className="btn-secondary"
+                style={{ marginTop: '10px' }}
+              >
+                📋 Copy Code
+              </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)